### DISCUSSION TITLE
CSV Feed - Shopware doesn't change the url to CDN Storage

### DISCUSSION THREAD

**Discussion by ghost** [FriendsOfShopware/FroshPlatformThumbnailProcessor#74] [Q&A] [answered]
Hi folks,

I am using the shopware productcompare feature to create a data CSV Feed for hood.de

Before the installation of bunny cdn the url was logical wise like this. https://www..moebelfaktor.de/media/74/18/7e/1662478621/CH02240BLNA.jpg

Now after all the changes it is like defined in the processor plugin (plus i created a CNAME for moebelfaktor.de (media.moebelfaktor.de)

Now after exporting the data the url doesn't automatically change to the storage url. Where is my mind abroad ... IS a CDN generally deactivating the feed image relations?

thanks a lot in advance i am lost. 

Best regards mark

**Comment by ghost** [comment, marked as answer]
Solved nevermind!

**Reply by tinect** [reply]
Could you please share the issue and solution, so anyone with the same issue could find a solution?

**Reply by ghost** [reply]
i first thought of descripe the solution. but i was just blind and there was no error!
